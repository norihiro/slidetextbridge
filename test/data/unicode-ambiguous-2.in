6Ωの抵抗と3Ωの抵抗を並列に接続すると、2Ωになります。
