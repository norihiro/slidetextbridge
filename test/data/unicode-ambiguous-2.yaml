steps:
  - type: stdin

  - type: linebreak
    ambiguous_char_width: 2
    split_long_line: 16

  - type: stdout
    page_delimiter: "\n"
