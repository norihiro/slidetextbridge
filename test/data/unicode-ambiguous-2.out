6Ωの抵抗と3Ωの
抵抗を並列に接続
すると、2Ωにな
ります。
